viral/response_profiles.py

```
@@ -0,0 +1,179 @@
+
+    save_path = Path(
+        SERVER_PATH
+        / "viral_caches"
+        / "response_profiles"
+        / f"{mouse_name}_{date}_rewarded_{rewarded}.npy",
+    )
+    signal = np.load(save_path)
+    mean = gaussian_filter1d(np.nanmean(signal, axis=0), sigma=1)
+    print("Mean shape", mean.shape)
+    if np.any(np.isnan(mean)):
+        warnings.warn(f"NaNs found in mean response for {mouse_name} {date}")
```

Ruff (0.14.14)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)